content/applications/websites/website/web_design/building_blocks.rst

```diff
@@ -0,0 +1,133 @@
+those elements automatically update to reflect the changes made to the selected preset.
+
+.. seealso::
+   :doc:`Website themes <themes>`
+
+Layout: Grid vs Cols
+====================
+
+Go to the :guilabel:`Customize` tab and then change the :guilabel:`Layout` of a selected element to
+:guilabel:`Grid` or :guilabel:`Cols` in the :guilabel:`Banner` section.
+
+Grid
+----
+
+The :guilabel:`Grid` layout allows you to reposition and resize elements, such as images or text, by
+dragging and dropping them to the desired place.
+
+.. image:: building_blocks/grid-layout.png
+   :alt: When the grid layout is selected, choose an image and drag and drop it where needed.
+
+.. tip::
+   Position images behind the text by using the above/below icons.
+
+   .. image:: building_blocks/superimpose-images-to-text.png
+      :alt: Use the above/below icons to superimpose images to text.
+
+Cols
+----
+
+The :guilabel:`Cols` feature is available for some blocks like :guilabel:`Columns` or
+:guilabel:`Image - Text` blocks. By choosing the :guilabel:`Cols` :guilabel:`Layout`, you can
+determine the number of elements per line within the block.
+
+.. image:: building_blocks/elements-per-line.png
+   :alt: Set the number of elements per line in the block.
+
+.. note::
+   In the mobile view, only one element per column is used by default for better visibility.
+   However, you can adjust this setting without impacting the desktop view.
+
+Duplicating a building block
+============================
+
+You can replicate a building block by clicking on the duplicate icon. Once duplicated, the new block
+appears on your website beneath the original one.
+
+.. image:: building_blocks/duplicate-container.png
+   :alt: Replicating a building block
+
+Re-ordering building blocks
+===========================
+
+To re-order the building blocks on your site, select the block you want to move and click the up
+arrow to move it before the previous section or click the down arrow to move it after the following
+section.
+
+You can also use the drag-and-drop icon to move your block manually.
+
+.. image:: building_blocks/re-ordering-blocks.png
+   :alt: Re-order building blocks by using the up and down arrow icons.
+
+Show/Hide on mobile
+===================
+
+You can hide specific elements of your website for mobile devices. To do so, select the element
+to hide, and in the :guilabel:`Customize` tab, scroll down to :guilabel:`Visibility`, and click the
+:guilabel:`Show/Hide on Mobile` icons.
+
+.. image:: building_blocks/show-hide-on-mobile.png
+   :alt: Click the "show/hide on mobile" icons to show or hide some elements on mobile.
+
+You can decide to show/hide with :guilabel:`No condition` or
+:guilabel:`Conditionally`, depending on :guilabel:`Country` (based on Geo IP), :guilabel:`Languages`
+(for multi-language websites), :guilabel:`UTM Campaign`, :guilabel:`UTM Medium`,
+:guilabel:`UTM Source`, or :guilabel:`Users`.
```

Comment:
```suggestion
You can also show or hide building blocks depending on specific conditions.
To do so, click :guilabel:`No condition` and select :guilabel:`Conditionally` instead. Next, under one of the available,  conditions, select if the building block should be :guilabel:`Visible for` or :guilabel:`Hidden for` specific records.
```
The available conditions depend on your DB, not everyone uses multiple languages or UTM campaigns. I think describing the general process is more interesting, especially if you don't describe what are the options.